converter/src/main/java/gov/cms/qpp/conversion/validate/IaSectionValidator.java

```diff
@@ -20,7 +20,17 @@
 	protected void internalValidateSingleNode(Node node) {
 		check(node)
 				.childMinimum(ErrorCode.IA_SECTION_MISSING_IA_MEASURE, 1, TemplateId.IA_MEASURE)
-				.childExact(ErrorCode.IA_SECTION_MISSING_REPORTING_PARAM, 1, TemplateId.REPORTING_PARAMETERS_ACT)
-				.onlyHasChildren(ErrorCode.IA_SECTION_WRONG_CHILD, TemplateId.IA_MEASURE, TemplateId.REPORTING_PARAMETERS_ACT);
+				.onlyHasChildren(ErrorCode.IA_SECTION_WRONG_CHILD, TemplateId.IA_MEASURE, TemplateId.REPORTING_PARAMETERS_ACT)
+			    .noParentChildDuplications(ErrorCode.IA_SECTION_IA_MEASURE_DUPLICATE_REPORTING_PARAM,
+				    TemplateId.REPORTING_PARAMETERS_ACT, TemplateId.IA_MEASURE);
+
+		if (node.getChildNodes(TemplateId.REPORTING_PARAMETERS_ACT).count() > 0) {
```

Comment:
Using findAny().isPresent() could be better here -- count() makes the stream do the full computation

Reply:
👍  will change.